### ISSUE TITLE
Error using Vagrant Up

### ISSUE THREAD

**Issue by toe** [scotch-io/scotch-box-build-scripts#1]
Hello,
I'm playing around with using build scripts with my Scotch Box and when I ran 'vagrant up' I received the following error. 

```
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Box 'bento/ubuntu-16.04' could not be found. Attempting to find and install...
    default: Box Provider: virtualbox
    default: Box Version: >= 0
The box 'bento/ubuntu-16.04' could not be found or
could not be accessed in the remote catalog. If this is a private
box on HashiCorp's Atlas, please verify you're logged in via
`vagrant login`. Also, please double-check the name. The expanded
URL and error message are shown below:

URL: ["https://atlas.hashicorp.com/bento/ubuntu-16.04"]
Error:
```
Any idea why this is happening and is there a workaround? 

Thank you,
-p

**Comment by whatnickcodes** [comment]
What version of Vagrant do you have?

This is the box we use (you can see it exists): https://atlas.hashicorp.com/bento/boxes/ubuntu-16.04

Maybe just specify the URL too: https://atlas.hashicorp.com/bento/boxes/ubuntu-16.04/versions/2.3.5/providers/virtualbox.box

**Comment by whatnickcodes** [comment]
Looks like might be a Vagrant bug: https://github.com/roots/roots-example-project.com/issues/54

**Comment by toe** [comment]
Here's my Vagrant config


```
# -*- mode: ruby -*-
# vi: set ft=ruby :

Vagrant.configure("2") do |config|

    config.vm.box = "bento/ubuntu-16.04"

    # Change this to the name of your project.
    # single string, no spaces, numbers, or special chars.
    config.vm.hostname = "test"

    # I normally never change anything, but the IP.
    # Depending if you have multiple vagrant boxes.
    config.vm.network "forwarded_port", guest: 80, host: 8080
    config.vm.network "private_network", ip: "192.168.33.10"

    # Set up your dev direcotry within a www/src direcory, but only map to www.
    # Everyhting else will happen naturally.
    config.vm.synced_folder "./www", "/var/www", :mount_options => ["dmode=777", "fmode=666"]

    config.ssh.insert_key = false

    config.vm.provision "shell", path: "install.sh", privileged: false

end
```

**Comment by toe** [comment]
I'll take a look at the defect and find a workaround. 

Thank you,
-p

**Comment by toe** [comment]
Fixed: I upgraded to vagrant 1.9.5, the most recent version.

**Comment by whatnickcodes** [comment]
That would do it! It's "Step 0" in the docs. https://box.scotch.io/members/
